tools/cpp/osx_cc_wrapper.sh.tpl

```diff
@@ -27,7 +27,7 @@
 #
 set -eu
 
-INSTALL_NAME_TOOL="/usr/bin/install_name_tool"
+INSTALL_NAME_TOOL="/usr/bin/xcrun install_name_tool"
```

Comment:
i think it would be better to inline this now that it has a space to be more careful around folks accidentally quoting this